database/migrations/2023_04_25_000000_add_source_to_poly_lines.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void {
```

Comment:
do we maybe want to add some kind of "parent" reference, so that we can find the corresponding _original_ polyline and won't create many redundant polylines?